helm/calico-capz/templates/capz-calico.yaml

```diff
@@ -0,0 +1,1222 @@
+---
+# Source: calico/templates/calico-config.yaml
+# This ConfigMap is used to configure a self-hosted Calico installation.
+kind: ConfigMap
```

Comment:
Is the capz repo the right location for a calico helm chart? Is there an upstream chart we could use or contribute to?  

What is the plan to bring in upstream changes from calico?  This removes some of the automation, that helped keep the calico features upto date.

Reply:
My working plan is to start by proving out calico CNI + helm chart (in progress), after which I'll investigate if we can use the official upstream chart(s) to do this work.